tests/e2e/Makefile

```diff
@@ -33,8 +33,12 @@ help: ## Display this help.
 
 .PHONY: create_module_kubebuilder_project
 create_module_kubebuilder_project:
-	go test --tags=create_module_kubebuilder_project
+	go test ./create_module -run "Test_ModuleTemplate"
 
 .PHONY: create_module_module_config
 create_module_module_config:
-	go test --tags=create_module_module_config
+	go test ./create_module -run "Test_ModuleTemplate"
```

Comment:
What is the reason for having two make targets `create_module_kubebuilder_project` and  `create_module_module_config` that invoke the same command underneath?

Reply:
because in the GitHub action we have two in the matrix with these names